src/themes/shared/components/Dropdown/dropdown-styles.ts

```diff
@@ -0,0 +1,53 @@
+import {
+  ComponentSlotStylesPrepared,
+  TeamsThemeStylesProps,
+} from "@fluentui/react-northstar";
+
+const borderZero = {
+  borderTopWidth: "0",
+  borderRightWidth: "0",
+  borderBottomWidth: "0",
+  borderLeftWidth: "0",
+};
+
+export const dropdownStyles: ComponentSlotStylesPrepared<
+  NonNullable<Partial<TeamsThemeStylesProps["Dropdown"]>>,
+  any
+> = {
+  root: (componentStyleParameters) => {
+    const {
+      props,
+      theme: {
+        siteVariables: { colorScheme },
+      },
+    } = componentStyleParameters;
+    return {
+      borderRadius: "4px",
```

Comment:
Should we use border radius from siteVariables?

Reply:
It's an excellent question, I'll check. I don't think `siteVariables` declares any `borderRadius`, but maybe it's high time it does.